### PR TITLE
Update container API grouping

### PR DESCRIPTION
**Pull request by Steven-Chan**
connect #231

### CODE REVIEW

File: packages/skygear-core/lib/container.js [hunk 1]
```diff
@@ -673,77 +242,121 @@ export default class Container {
...
+     * @property {number} [timeoutOptions.response=60000] - maximum time to
+     * wait for an response (in milliseconds)
+     *
+     * @see http://visionmedia.github.io/superagent/#timeouts
+     */
+    this.timeoutOptions = {
+      response: 60000
+    };
+  }
+
```
Comment (rickmak) [inline packages/skygear-core/lib/container.js]: Please add esdocs annotation to `Container`. At least to make the doc navigtable, and we can fill in the detail laters.

File: packages/skygear-core/lib/database.js [hunk 2]
```diff
@@ -33,6 +33,10 @@ export default class Database {
     this._cacheResponse = true;
   }
 
+  get Record() {
+    return this.container.Record;
+  }
```
Comment (rickmak) [inline packages/skygear-core/lib/database.js]: I am in double the Record are accessible from boath BaseContainer and database container. Semm we should choose one. In this case, I will choose BaseContainer.

Comment (Steven-Chan) [inline packages/skygear-core/lib/database.js]: Well, what if this is for internal use? Like `User` and `publicDB` in `AuthContainer`. (I think these are two different cases, `User` can actually be imported directly, but `publicDB` cannot.)

Comment (rickmak) [inline packages/skygear-core/lib/database.js]: Mark it as private by annotation or prefix with an underscore?

**Comment by Steven-Chan** [comment]
@rickmak Updated

I've also made some extra changes on the skygear.relation on 333676d2753f6c74b096012603eef43d093966a1

File: example/index.html [hunk 3]
```diff
@@ -78,7 +78,7 @@
     <hr>
     <p><button onclick="logout();">Logout</button></p>
   </div>
-  <script src="../dist/bundle.js"></script>
+  <script src="../packages/skygear-core/dist/bundle.js"></script>
```
Comment (cheungpat) [inline example/index.html]: good catch

**Pull request merged**